docs/manage/configuration.md

```diff
@@ -26,29 +26,6 @@ Once the robot has a configuration, it caches it locally and can use the configu
 The robot checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
 All communication happens securely over HTTPS using secret tokens that are in a robot's configuration.
 
-{{< alert title="Tip" color="tip" >}}

```

Comment:
why are you removing this? I've just added it

Reply:
@npentrel Oh sorry, I think I got this confused while rebasing-- will add back in